pilot/cmd/pilot-agent/main.go

```diff
@@ -142,6 +143,19 @@ var (
 			role.TrustDomain = spiffe.GetTrustDomain()
 			log.Infof("Proxy role: %#v", role)
 
+			// Watch default cert locations if none were set
+			if len(tlsCertsToWatch) == 0 {
+				tlsCertsToWatch = []string{
+					path.Join(model.AuthCertsPath, model.CertChainFilename),
+					path.Join(model.AuthCertsPath, model.KeyFilename),
+					path.Join(model.AuthCertsPath, model.RootCertFilename),
+				}
+				if role.Type == model.Ingress {
```

Comment:
@rshriram mentioned that we probably don't need this. Can this be confirmed? if not I'd prefer to remove it in a separate PR in case it needs to be reverted.

Reply:
keep it for now